spec/integration/apply_spec.rb

```diff
@@ -38,6 +59,55 @@ def root_config
       run_command(uninstall, uri, config: config_data, inventory: inventory_data)
     end
 
+    context "when running against puppet 5 or puppet 6" do
```

Comment:
We should also clean up (uninstall) packages so we can re-run tests. Or document that docker images need to be recreated if you want to re-run tests (which seems like a bad experience).

Reply:
I guess testing install is not really an important part. We could make this part of building the images; or just document that tests will install to these agents initially, then re-use the installed agent.